pkg/cli/create.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kubebuilder/pkg/cli/internal"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+func (c cli) newCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Scaffold a Kubernetes API or webhook",
+		Long:  `Scaffold a Kubernetes API or webhook.`,
+	}
+	cmd.AddCommand(c.newCreateAPICmd())
+
+	if !internal.ConfiguredAndV1() {
+		cmd.AddCommand(c.newCreateWebhookCmd())
+	}
+
+	return cmd
+}
+
+func (c cli) newCreateAPICmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "api",
+		Short: "Scaffold a Kubernetes API",
+		Long: `Scaffold a Kubernetes API.
+`,
+		RunE: errCmdFunc(
+			fmt.Errorf("api subcommand requires an existing project"),
+		),
+	}
+
+	projectVersion, found := getProjectVersion()
+	if !found {
+		msg := `For project-specific information, run this command in the root directory of a
+project.
+`
+		cmd.Long = fmt.Sprintf("%s\n%s", cmd.Long, msg)
```

Comment:
I think we should return an error.

Reply:
I think you pointed to the wrong line or something. That replacement doesn't make sense.